src/NHibernate/Mapping/PersistentClass.cs

```diff
@@ -658,6 +658,11 @@ public virtual void AddProperty(Property p)
 		/// <value>The CacheConcurrencyStrategy used with the Cache.</value>
 		public abstract string CacheConcurrencyStrategy { get; set; }
 
+		/// <summary>
+		/// When <see cref="CacheConcurrencyStrategy"/> is "never" then UpdateTimestampsCache is not tracked
+		/// </summary>
+		public abstract bool HasUpdateTimestampsCache { get; }
```

Comment:
This is also a breaking change that prevents the PR to be included into the next minor version. I think it is better to avoid adding it and instead update the added line in `AbstractEntityPersister.cs` to:
```
hasUpdateTimestampsCache  = persistentClass.CacheConcurrencyStrategy != CacheFactory.Never;
```

Reply:
ok